libprecompiled/Precompiled.h

```diff
@@ -20,6 +20,7 @@
  */
 #pragma once
 
+#include "Common.h"
```

Comment:
only precompiled.cpp need this?

Reply:
because many Precompiled need getParamFunc?

Reply:
Yes, many Precompiled need `getParamFunc`, so I put `Common.h` into `Precompiled.h`.